chargeOclock/Source/Models/Client.swift

```diff
@@ -15,10 +15,15 @@ protocol ClientSettable {
 class Client {
 
 	var id: Int = 0
-	var name: String = ""
+	var isDirty: Bool = false
 
-	init(id: Int, name: String) {
-		self.id = id
+	var name: String = "" {
+		didSet {
+			isDirty = true
+		}
+	}
+
+	init(name: String) {
 		self.name = name
```

Comment:
What's the purpose of `isDirty` parameter, that was introduced during previous feedback. I thought it was intended to provide the information weather the `id` (then non-optional) was valid or not. So now that the `id` is optional there's no more need for the `isDirty` parameter I guess. Is that the case, or am I missing something? And how "cleaning" to `isDirty` was suposed to happen? How should `isDirty` be set to `false` in first place?